### ISSUE TITLE
Clicking Exit to Title Screen/Desktop has no prompt to save

### ISSUE THREAD

**Issue by ike709** [tsunamayo/Skywanderers#163]
Clicking one of the Exit buttons should really ask you if you want to save before exiting.

**Comment by tsunamayo** [comment]
Yes I agree I planned to do this, as an auto-save feature (auto-save will wait a bit more though).
